README.md

```diff
@@ -21,6 +21,13 @@ xhr({
 ```
 
 ## `var req = xhr(options, callback)`
+Using this invocation, `options` should include a `uri` property for the request.
```

Comment:
that's not entirely true, uri or url. It's weird but that's compatibility. Options are listed below, should be enough.